tests/Functional/src/com/microsoft/aad/adal/test/AuthenticationParamsTests.java

```diff
@@ -196,7 +196,7 @@ public void testParseResponseNegative() throws IllegalArgumentException,
                 AuthenticationParameters.AUTH_HEADER_INVALID_FORMAT);
 
         callParseResponseForException(
-                new HttpWebResponse(401, null, getHeader("WWW-Authenticate", "Bearer foo ")),
+                new HttpWebResponse(401, null, getHeader("WWW-Authenticate", "Bearer test ")),
```

Comment:
Okay, I just think it might confused people. Everything else looks good in my view.